### PR TITLE
[XGUARD-1318] members tests

### PR DESCRIPTION
**Pull request by SiamakSamie**

### CODE REVIEW

**Review by sebdesl — APPROVED** [review]

File: tests/Unit/Members/MembersTest.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        parent::setUp();
+        $this->seed();
+        $user = factory(User::class)->create();
+        Auth::setUser($user);
+    }
+
+    public function testAUserCanCreateSeveralMembers()
+    {
+
```
Comment (NicoChampagne) [inline tests/Unit/Members/MembersTest.php:32]: remove extra line

File: tests/Unit/Members/MembersTest.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+use Xguard\LaravelKanban\Actions\Members\DeleteMemberAction;
+use Xguard\LaravelKanban\Actions\Members\UpdateOrCreateMemberAction;
+use Xguard\LaravelKanban\Models\Board;
+use Xguard\LaravelKanban\Models\Employee;
+use Xguard\LaravelKanban\Models\Member;
+use Xguard\LaravelKanban\Models\Log;
+
+class MemberTest extends TestCase
+{
+
```
Comment (NicoChampagne) [inline tests/Unit/Members/MembersTest.php:19]: remove extra line

File: src/Actions/Members/UpdateOrCreateMemberAction.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+use Illuminate\Contracts\Routing\ResponseFactory;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+use Xguard\LaravelKanban\Models\Log;
+use Xguard\LaravelKanban\Models\Member;
+use Lorisleiva\Actions\Action;
+
+class UpdateOrCreateMemberAction extends Action
+{
+
```
Comment (NicoChampagne) [inline src/Actions/Members/UpdateOrCreateMemberAction.php:15]: remove extra line

File: src/Actions/Members/DeleteMemberAction.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+use Xguard\LaravelKanban\Models\Log;
+use Xguard\LaravelKanban\Models\Member;
+
+use Lorisleiva\Actions\Action;
+
+class DeleteMemberAction extends Action
+{
+
```
Comment (NicoChampagne) [inline src/Actions/Members/DeleteMemberAction.php:16]: remove extra line

File: src/Actions/Members/DeleteMemberAction.php [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    /**
+     * Execute the action and return a result.
+     *
+     * @return Application|ResponseFactory|Response
```
Comment (NicoChampagne) [inline src/Actions/Members/DeleteMemberAction.php:20]: these comments are noise

File: src/Actions/Members/DeleteMemberAction.php [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+     */
+    public function handle()
```
Comment (NicoChampagne) [inline src/Actions/Members/DeleteMemberAction.php:22]: you can add return type at the end of handle ie: public function handle() : Response

File: src/Actions/Members/UpdateOrCreateMemberAction.php [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    /**
+     * Execute the action and return a result.
+     *
+     * @return Application|ResponseFactory|Response
+     */
```
Comment (NicoChampagne) [inline src/Actions/Members/UpdateOrCreateMemberAction.php:20]: same as other file comments

**Comment by NicoChampagne** [comment]
rebase

**Pull request closed without merging**